scripts/broker.csx

```diff
@@ -0,0 +1,115 @@
+#r "System.Data"
+#r "../src/packages/RabbitMQ.Client.4.1.1/lib/net451/RabbitMQ.Client.dll"
+
+using System.Data.Common;
+using System.Net;
+using RabbitMQ.Client;
+
+public void DeleteVirtualHost()
+{
+    try
+    {
+        CreateVirtualHostRequest("DELETE", GetBroker()).GetResponse().Dispose();
+    }
+    catch (WebException ex) when ((ex?.Response as HttpWebResponse)?.StatusCode == HttpStatusCode.NotFound)
+    {
+    }
+}
+
+public void CreateVirtualHost() => CreateVirtualHostRequest("PUT", GetBroker()).GetResponse().Dispose();
+
+public void AddUserToVirtualHost() => CreateUserPermissionRequest("PUT", GetBroker()).GetResponse().Dispose();
+
+public Broker GetBroker()
+{
+    var connectionStringBuilder = new DbConnectionStringBuilder
+    {
+        ConnectionString = Environment.GetEnvironmentVariable("RabbitMQTransport.ConnectionString"),
```

Comment:
> If it was many items, I'd prefer to keep the trailing comma on the last line so that all lines have the same format and can be moved around easily.

I can understand that, though personally the comma still bothers me too much to be able to take advantage of that convenience. 😄